Manifests/ManifestsApple/com.apple.loginwindow.plist

```diff
@@ -608,6 +609,22 @@ Availability: Available in macOS 10.13 and later.</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<true/>
+			<key>pfm_description</key>
+			<string>Reopens windows that were open at time of logout</string>
+			<key>pfm_description_reference</key>
+			<string></string>
+			<key>pfm_documentation_source</key>
```

Comment:
No reason to leave in keys with empty strings.

Reply:
Must admit, I never even knew the `LoginwindowLaunchesRelaunchApps` key existed so yeah, `TALLogoutSavesState` does work independently. I'll fix the empty keys.